domains/certificates/Query.API/API.UnitTests/DataSyncSyncer/DataSyncServiceTest.cs

```diff
@@ -0,0 +1,164 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using API.DataSyncSyncer.Dto;
+using API.MasterDataService;
+using CertificateEvents.Primitives;
+using IntegrationEvents;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Newtonsoft.Json;
+using RichardSzalay.MockHttp;
```

Comment:
Why use both Moq and MockHttp?

Reply:
Moq is used to mock the DI classes and mockHttp is used to mock the httpClient.
as described here https://github.com/Energinet-DataHub/energy-origin/blob/main/doc/architecture/adr/0010-nuget-mockhttp.md